source/dpp/translation/macro_.d

```diff
@@ -3,6 +3,7 @@ module dpp.translation.macro_;
 
 import dpp.from;
 
+import clang: Token;
```

Comment:
Please don't introduce global imports.

Reply:
Token is used almost everywhere, so I thought this would be better just as global import. Is there a reason why this shouldn't be a global import?